src/SFA.DAS.EmployerCommitmentsV2.Web/Mappers/Cohort/ReviewRequestViewModelMapper.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using SFA.DAS.CommitmentsV2.Api.Client;
+using SFA.DAS.CommitmentsV2.Api.Types.Requests;
+using SFA.DAS.CommitmentsV2.Shared.Interfaces;
+using SFA.DAS.CommitmentsV2.Types;
+using SFA.DAS.EmployerCommitmentsV2.Web.Models.Cohort;
+using SFA.DAS.EmployerCommitmentsV2.Web.Extensions;
+using SFA.DAS.EmployerUrlHelper;
+using SFA.DAS.Encoding;
+
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Mappers.Cohort
+{
+    public class ReviewRequestViewModelMapper : IMapper<ReviewRequest, ReviewViewModel>
+    {
+        private readonly IEncodingService _encodingService;
+        private readonly ICommitmentsApiClient _commitmentsApiClient;
+        private readonly ILinkGenerator _linkGenerator;
+
+        public ReviewRequestViewModelMapper(ICommitmentsApiClient commitmentApiClient, IEncodingService encodingSummary, ILinkGenerator linkGenerator)
+        {
+            _encodingService = encodingSummary;
+            _commitmentsApiClient = commitmentApiClient;
+            _linkGenerator = linkGenerator;
+        }
+
+        public async Task<ReviewViewModel> Map(ReviewRequest source)
+        {
+            var cohortsResponse = await _commitmentsApiClient.GetCohorts(new GetCohortsRequest { AccountId = source.AccountId });
+
+            var reviewViewModel =  new ReviewViewModel
+            {
+                AccountHashedId = source.AccountHashedId,
+                BackLinkUrl = _linkGenerator.Cohorts(source.AccountHashedId),
+                CohortSummary = cohortsResponse.Cohorts
+                .Where(x => x.WithParty == Party.Employer && !x.IsDraft)
```

Comment:
As with Draft, we could use an extension method to determine cohort "status"

Reply:
Created an extension function.